oio/cli/storage/account.py

```diff
@@ -143,3 +143,43 @@ def take_action(self, parsed_args):
             account=parsed_args.account,
             properties=parsed_args.property
         )
+
+
+class ListAccount(lister.Lister):
+    """List Account"""
+
+    log = logging.getLogger(__name__ + '.ListAccount')
+
+    def get_parser(self, prog_name):
+        parser = super(ListAccount, self).get_parser(prog_name)
+        parser.add_argument(
+            '--full',
+            dest='full_listing',
+            default=False,
+            help='Full listing',
+            action="store_true"
+        )
+        return parser
+
+    def take_action(self, parsed_args):
+        self.log.debug('take_action(%s)', parsed_args)
+
+        data = self.app.client_manager.storage.account_list()
+        accounts = ((e,) for e in data)
+
+        if parsed_args.full_listing:
+            columns = ('Name', 'bytes', 'containers', 'ctime', 'metadata',
```

Comment:
I would have displayed `metadata` at the end because it's the least important, and `objects` just after `containers`.